src/main/java/org/fundacionjala/at15/katas/langtonant/mauricio/Ant.java

```diff
@@ -0,0 +1,151 @@
+
+package org.fundacionjala.at15.katas.langtonant.mauricio;
+
+public class Ant {
+    private int posX;
+    private int posY;
+    private Grid grid;
+    private String direction;
+    private boolean finished;
+
+    public Ant(int x, int y, Grid grid, String direction) {
+        posX = x;
+        posY = y;
+        this.grid = grid;
+        this.direction = direction;
+        finished = false; // the game is ready
+    }
+
+    public void nextStep() {
+        switch (direction) {
+            case "up":
+                // System.out.println("UP");
```

Comment:
If not needed, lets remove commented code

Reply:
Thanks for the corrections Jose, i'm going to work on that now